app/templates/suppliers/companies_house_number.html

```diff
@@ -28,12 +28,13 @@
 
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="hint">
-        <p>This is a unique 8-character number given to your organisation by Companies House.</p>
-        <p><a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a></p>
-      </div>
       <form method="POST" action="{{ url_for('.companies_house_number') }}">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {% set question_advice =
+            'This is a unique 8-character number given to your organisation by Companies House.
+            \n\n<a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a>
```

Comment:
Both methods are a bit odd because of the things YAML implies from newlines in certain places.

The point of @pcraig3 's change seems to be to make it more explicit how the paragraphs will wrap the given text. That being so I'd plump for the method using literal `\n`s.

We should try and work with this method and see if it removes some of the confusion (or causes more?).